enterprise/backend/src/metabase_enterprise/workspaces/models/workspace_dependency.clj

```diff
@@ -0,0 +1,14 @@
+(ns metabase-enterprise.workspaces.models.workspace-dependency
+  "Model for WorkspaceDependency - edges between workspace transforms and their dependencies."
+  (:require
+   [metabase.models.interface :as mi]
+   [methodical.core :as methodical]
+   [toucan2.core :as t2]))
+
+(methodical/defmethod t2/table-name :model/WorkspaceDependency [_model] :workspace_dependency)
+
+(derive :model/WorkspaceDependency :metabase/model)
+
+(t2/deftransforms :model/WorkspaceDependency
+  {:from_entity_type mi/transform-keyword
+   :to_entity_type   mi/transform-keyword})
```

Comment:
Are the entity ids here always ref ids? Maybe we want to trim them too.

Now that I think about it, it would be nice if we stored dependencies for the enclosed elements also, and those will have integer ids 🤔